### ISSUE TITLE
L2正则化  - 权重衰减 - weight decay

### ISSUE THREAD

**Issue by junxnone** [junxnone/tech-io#562]
# Reference
- [ ] [权重衰减（weight decay）与学习率衰减（learning rate decay）](https://blog.csdn.net/program_developer/article/details/80867468)
